src/neuroconv/datainterfaces/behavior/audio/audiointerface.py

```diff
@@ -0,0 +1,203 @@
+from pathlib import Path
+from typing import Optional
+from warnings import warn
+
+from hdmf.backends.hdf5 import H5DataIO
+from natsort import natsorted
+from scipy.io.wavfile import read
+
+from neuroconv.basedatainterface import BaseDataInterface
+from neuroconv.datainterfaces.behavior.video.videodatainterface import _check_duplicates
+from neuroconv.tools.hdmf import SliceableDataChunkIterator
+from neuroconv.tools.nwb_helpers import (
+    make_or_load_nwbfile,
+)
+from neuroconv.utils import (
+    get_schema_from_hdmf_class,
+    get_base_schema,
+    OptionalFilePathType,
+)
+from pynwb import NWBFile, TimeSeries
+
+from ndx_sound import AcousticWaveformSeries
+
+
+class AudioInterface(BaseDataInterface):
+    """Data interface for writing acoustic recordings to an NWB file."""
+
+    def __init__(self, file_paths: list, verbose: bool = False):
+        """
+        Create the interface for writing acoustic recordings as AcousticWaveformSeries.
+
+        Parameters
+        ----------
+        file_paths: list of FilePathTypes
+            The file paths to the audio recordings in sorted, consecutive order.
+        """
+        suffixes = [suffix for file_path in file_paths for suffix in Path(file_path).suffixes]
+        format_is_not_supported = [suffix for suffix in suffixes if suffix not in [".wav"]]
+        if format_is_not_supported:
+            raise ValueError(
+                "The currently supported file format for audio is WAV file. "
+                f"Some of the provided files does not match this format: {format_is_not_supported}."
+            )
+        self.verbose = verbose
+        super().__init__(file_paths=natsorted(file_paths))
```

Comment:
It's always a question of whether or not to use natsorted at this level. Does the VideoInterface do that? I'd lead more towards specifying whatever order the user wants, and encouraging them to use natsort through documentation if they have a naming scheme that actually follows their chronological timing